cilantro_audit/view_audit_templates.py

```diff
@@ -0,0 +1,55 @@
+import kivy
+from kivy.app import App
+from kivy.clock import Clock
+from kivy.uix.button import Button
+from kivy.uix.screenmanager import Screen
+from kivy.uix.screenmanager import ScreenManager
+from audit_template import AuditTemplate
+from mongoengine import connect
+
+# Required version
+kivy.require('1.11.1')
+
+connect("toost")
```

Comment:
Everything in the app should be using the `PROD_DB` constant.

Reply:
My bad, didn't realize PROD_DB was a thing